pallets/ds-maps/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+authors = ['Mixbytes <https://github.com/mixbytes/dsky-substrate-template>']
+description = 'FRAME pallet for handling city maps, runtime logic.'
+edition = '2018'
+homepage = 'https://substrate.dev'
+license = 'Apache-2.0'
+name = 'pallet-ds-maps'
+repository = 'https://github.com/mixbytes/dsky-substrate-template'
+version = '2.0.0'
+
+[package.metadata.docs.rs]
+targets = ['x86_64-unknown-linux-gnu']
+
+# alias "parity-scale-code" to "codec"
+[dependencies.codec]
+default-features = false
+features = ['derive']
+package = 'parity-scale-codec'
+version = '1.3.4'
+
+[dependencies]
+frame-support = { default-features = false, version = '2.0.0' }
+frame-system = { default-features = false, version = '2.0.0' }
+pallet-timestamp = { default-features = false, version = '2.0.0' }
+#pallet-balances = { default-features = false, version = '2.0.0' }
+serde = { default-features = false, version = '1.0.116', optional=true }
+pallet-transaction-payment = { default-features = false, version = '2.0.0', optional=true }
```

Comment:
This crate has nothing to do with payments/balances, so why it depends on such things? Pls remove unnecessary template garbage everywhere.